asteroid/masknn/blocks.py

```diff
@@ -460,3 +504,114 @@ def forward(self, input_data):
         mask_out = self.mask_layer(out)
         mask_out = mask_out.view(batches, self.n_src, self.input_dim, seq_cnt)
         return projection_final, mask_out
+
+
+class TwoStepTDCN(nn.Module):
```

Comment:
Architectures in `blocks.py` do not include encoders and decoders, it makes everything inside the the source code more modular and reusable.